### DISCUSSION TITLE
wikipedia_zh_all_maxi_2024-04.zim file lacks some key information.

### DISCUSSION THREAD

**Discussion by fishergz** [kiwix/kiwix-android#3903] [General]
kiwix
![1424349316](https://github.com/kiwix/kiwix-android/assets/22692819/fd6aaf87-5cff-43f2-8c38-4c591e9b530e)

wiki
![732178238](https://github.com/kiwix/kiwix-android/assets/22692819/99fe52fe-2d2a-4dc2-8c71-4b2a6900ddb6)

This problem exists in all Chinese zim files

**Comment by kelson42** [comment]
This is actually a duplicate of github

**Reply by fishergz** [reply]
sorry,I don't quite understand "duplicate of github"

**Reply by kelson42** [reply]
This is already tracked somewhere else

**Reply by fishergz** [reply]
Okay, thank you for informing me, I will wait patiently. Thank you for your hard work.
